app/assets/stylesheets/restrooms/new.scss

```diff
@@ -1,3 +1,7 @@
+.form-control{
```

Comment:
I think we can do this globally and not just in "new". The location lookup also displays gray test.